automap/src/comm_layer/pcp_pmp_common/windows_specific.rs

```diff
@@ -0,0 +1,401 @@
+   IPv4 Address. . . . . . . . . . . : 10.0.2.15(Preferred)
+   Subnet Mask . . . . . . . . . . . : 255.255.255.0
+   Lease Obtained. . . . . . . . . . : Sunday, February 14, 2021 5:21:59 PM
+   Lease Expires . . . . . . . . . . : Monday, February 15, 2021 2:45:20 PM
+   Default Gateway . . . . . . . . . : 10.0.2.0
+   DHCP Server . . . . . . . . . . . : 10.0.2.3
+   DHCPv6 IAID . . . . . . . . . . . : 101187623
+   DHCPv6 Client DUID. . . . . . . . : 00-01-00-01-26-49-5A-82-08-00-27-4B-EB-0D
+   DNS Servers . . . . . . . . . . . : 192.168.0.1
+   NetBIOS over Tcpip. . . . . . . . : Enabled
+";
+        let find_routers_command = FindRoutersCommandMock::new(Ok(&route_n_output));
+
+        let result = windows_find_routers(&find_routers_command).unwrap();
+
+        assert_eq!(
+            result,
+            vec![
+                IpAddr::from_str("10.0.2.2").unwrap(),
+                IpAddr::from_str("10.0.2.0").unwrap(),
+            ]
+        )
+    }
+
+    #[test]
+    fn find_routers_works_on_another_specific_machine() {
+        let route_n_output = "
+Windows IP Configuration
+
+   Host Name . . . . . . . . . . . . : DESKTOP
+   Primary Dns Suffix  . . . . . . . :
+   Node Type . . . . . . . . . . . . : Hybrid
+   IP Routing Enabled. . . . . . . . : No
+   WINS Proxy Enabled. . . . . . . . : No
+
+Ethernet adapter Ethernet 3:
+
+   Connection-specific DNS Suffix  . :
+   Description . . . . . . . . . . . : VirtualBox Host-Only Ethernet Adapter
+   DHCP Enabled. . . . . . . . . . . : No
+   Autoconfiguration Enabled . . . . : Yes
+   Link-local IPv6 Address . . . . . : fi60::3011:4121:5c3b:f131%11(Preferred)
+   IPv4 Address. . . . . . . . . . . : 192.168.56.1(Preferred)
+   Subnet Mask . . . . . . . . . . . : 255.255.255.0
+   Default Gateway . . . . . . . . . :
+   NetBIOS over Tcpip. . . . . . . . : Enabled
+
+Wireless LAN adapter Local Network* 4:
+
+   Media State . . . . . . . . . . . : Media disconnected
+   Connection-specific DNS Suffix  . :
+
+Wireless LAN adapter Local Network* 9:
+
+   Media State . . . . . . . . . . . : Media disconnected
+   Connection-specific DNS Suffix  . :
+
+Ethernet adapter Ethernet 2:
+
+   Connection-specific DNS Suffix  . : domain
+   DHCP Enabled. . . . . . . . . . . : Yes
+   Autoconfiguration Enabled . . . . : Yes
+   Link-local IPv6 Address . . . . . : fd50::ed11:2c61:6111:f02e%10(Preferred)
+   IPv4 Address. . . . . . . . . . . : 192.168.10.10(Preferred)
+   Subnet Mask . . . . . . . . . . . : 255.255.255.0
+   Default Gateway . . . . . . . . . : 192.168.10.5
+   DHCP Server . . . . . . . . . . . : 192.168.1.1
+";
+        let find_routers_command = FindRoutersCommandMock::new(Ok(&route_n_output));
+
+        let result = windows_find_routers(&find_routers_command).unwrap();
+
+        assert_eq!(result, vec![IpAddr::from_str("192.168.10.5").unwrap()])
+    }
+
+    #[test]
+    fn find_routers_works_on_galactic_overlords_machine() {
```

Comment:
Please try also here to describe what is different from the above. Maybe it can read from a comment.